deployment/GP03FLMA_Deploy.csv

```diff
@@ -1,127 +1,127 @@
+MV1404,,TN323,,GP03FLMA-RIS01-05-FLORTD000,2,1,2014-06-18T05:56:00,2015-06-05T14:53:00,CGMGP-03FLMA-00002,,CGINS-FLORTD-01116,49.9775,-144.2463,,28,4127,
+TN323,,RB1605,,GP03FLMA-RIM01-00-SIOENG000,3,1,2015-06-06T22:40:00,2016-06-28T19:10:00,CGMGP-03FLMA-00003,,CGCON-SIOCON-12365,49.97667,-144.24617,,1504,4127,
+TN323,,RB1605,,GP03FLMA-RIM01-02-ADCPSL003,3,1,2015-06-06T22:40:00,2016-06-28T19:10:00,CGMGP-03FLMA-00003,,CGINS-ADCPSL-21393,49.97667,-144.24617,,498,4127,
+TN323,,RB1605,,GP03FLMA-RIM01-02-CTDMOG040,3,1,2015-06-06T22:40:00,2016-06-28T16:02:00,CGMGP-03FLMA-00003,,CGINS-CTDMOG-10219,49.97667,-144.24617,,28,4127,
+TN323,,RB1605,,GP03FLMA-RIM01-02-CTDMOG041,3,1,2015-06-06T22:40:00,2016-06-28T16:02:00,CGMGP-03FLMA-00003,,CGINS-CTDMOG-12411,49.97667,-144.24617,,38,4127,
+TN323,,RB1605,,GP03FLMA-RIM01-02-CTDMOG042,3,1,2015-06-06T22:40:00,2016-06-28T16:02:00,CGMGP-03FLMA-00003,,CGINS-CTDMOG-10224,49.97667,-144.24617,,58,4127,
+TN323,,RB1605,,GP03FLMA-RIM01-02-CTDMOG043,3,1,2015-06-06T22:40:00,2016-06-28T16:02:00,CGMGP-03FLMA-00003,,CGINS-CTDMOG-10220,49.97667,-144.24617,,88,4127,
+TN323,,RB1605,,GP03FLMA-RIM01-02-CTDMOG044,3,1,2015-06-06T22:40:00,2016-06-28T16:02:00,CGMGP-03FLMA-00003,,CGINS-CTDMOG-10221,49.97667,-144.24617,,128,4127,
+TN323,,RB1605,,GP03FLMA-RIM01-02-CTDMOG045,3,1,2015-06-06T22:40:00,2016-06-28T16:02:00,CGMGP-03FLMA-00003,,CGINS-CTDMOG-10215,49.97667,-144.24617,,178,4127,
+TN323,,RB1605,,GP03FLMA-RIM01-02-CTDMOG046,3,1,2015-06-06T22:40:00,2016-06-28T16:02:00,CGMGP-03FLMA-00003,,CGINS-CTDMOG-10252,49.97667,-144.24617,,248,4127,
+TN323,,RB1605,,GP03FLMA-RIM01-02-CTDMOG047,3,1,2015-06-06T22:40:00,2016-06-28T16:02:00,CGMGP-03FLMA-00003,,CGINS-CTDMOG-10217,49.97667,-144.24617,,348,4127,
+TN323,,RB1605,,GP03FLMA-RIM01-02-CTDMOG048,3,1,2015-06-06T22:40:00,2016-06-28T19:10:00,CGMGP-03FLMA-00003,,CGINS-CTDMOG-10263,49.97667,-144.24617,,498,4127,
+TN323,,RB1605,,GP03FLMA-RIM01-02-CTDMOH049,3,1,2015-06-06T22:40:00,2016-06-28T19:10:00,CGMGP-03FLMA-00003,,CGINS-CTDMOH-10226,49.97667,-144.24617,,748,4127,
+TN323,,RB1605,,GP03FLMA-RIM01-02-CTDMOH050,3,1,2015-06-06T22:40:00,2016-06-28T19:10:00,CGMGP-03FLMA-00003,,CGINS-CTDMOH-10229,49.97667,-144.24617,,998,4127,
+TN323,,RB1605,,GP03FLMA-RIM01-02-CTDMOH051,3,1,2015-06-06T22:40:00,2016-06-28T19:10:00,CGMGP-03FLMA-00003,,CGINS-CTDMOH-11601,49.97667,-144.24617,,1498,4127,
+TN323,,RB1605,,GP03FLMA-RIS01-00-SIOENG000,3,1,2015-06-06T22:40:00,2016-06-28T16:02:00,CGMGP-03FLMA-00003,,CGCON-SIOCON-12368,49.97667,-144.24617,,28,4127,
+TN323,,RB1605,,GP03FLMA-RIS01-03-DOSTAD000,3,1,2015-06-06T22:40:00,2016-06-28T16:02:00,CGMGP-03FLMA-00003,,CGINS-DOSTAD-00394,49.97667,-144.24617,,28,4127,
+TN323,,RB1605,,GP03FLMA-RIS01-04-PHSENF000,3,1,2015-06-06T22:40:00,2016-06-28T16:02:00,CGMGP-03FLMA-00003,,CGINS-PHSENF-P0080,49.97667,-144.24617,,28,4127,
+TN323,,RB1605,,GP03FLMA-RIS01-05-FLORTD000,3,1,2015-06-06T22:40:00,2016-06-28T16:02:00,CGMGP-03FLMA-00003,,CGINS-FLORTD-01216,49.97667,-144.24617,,28,4127,

```

Comment:
Deployment 3 – Lower release was fired at 18:10 UTC and upper release was fired at 15:02 (per Cruise report appendix and data).